packages/pwa-buildpack/package.json

```diff
@@ -93,17 +95,19 @@
     "babel-loader": "~8.0.5",
     "compression": "~1.7.4",
     "css-loader": "~2.1.1",
+    "postcss": "~8.3.6",
+    "postcss-loader": "~4.3.0",
     "terser-webpack-plugin": "~1.2.3",
     "webpack": "~4.46.0",
     "workbox-webpack-plugin": "~6.0.2"
   },
-  "resolutions": {
-    "workbox-build": "~6.0.2"
-  },
   "engines": {
     "node": ">=10.x",
     "yarn": ">=1.12.0"
   },
+  "resolutions": {
```

Comment:
If you are moving this you may as well remove this work-around. The issue was fixed in 6.2.2 see
https://github.com/GoogleChrome/workbox/issues/2904